.ci/get-latest-version.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+## This script prints the Version for go.elastic.co/apm/v2 in the stdout
+
+set -eo pipefail
+
+go list -json -u -m go.elastic.co/apm/v2 | jq -r 'try (.Update.Version) // .Version'
```

Comment:
@kuisathaverat's suggestion looks better, but for future reference I think this would do what you need, and be simpler:

`go list -f '{{.Version}}' -m go.elastic.co/apm/v2@latest`